Embulk 0.8.28 and 0.8.29 doesn't work Liquid {% include %}.

## Summary

Embulk 0.8.29 doesn't work liquid.  {% include %}
The following configuration works fine on 0.8.18, but It doesn't work on 0.8.19.

## Expect behavior

* Embulk 0.8.18

```
embulk preview config.yml.liquid
2017-08-10 12:53:43.210 +0900: Embulk v0.8.18
2017-08-10 12:53:44.502 +0900 [INFO] (0001:preview): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 12:53:44.506 +0900 [INFO] (0001:preview): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
+---------+--------------+-------------------------+-------------------------+----------------------------+
| id:long | account:long |          time:timestamp |      purchase:timestamp |             comment:string |
+---------+--------------+-------------------------+-------------------------+----------------------------+
|       1 |       32,864 | 2015-01-27 19:23:49 UTC | 2015-01-27 00:00:00 UTC |                     embulk |
|       2 |       14,824 | 2015-01-27 19:01:23 UTC | 2015-01-27 00:00:00 UTC |               embulk jruby |
|       3 |       27,559 | 2015-01-28 02:20:02 UTC | 2015-01-28 00:00:00 UTC | Embulk "csv" parser plugin |
|       4 |       11,270 | 2015-01-29 11:54:36 UTC | 2015-01-29 00:00:00 UTC |                            |
+---------+--------------+-------------------------+-------------------------+----------------------------+
```

### Actual behavior

* Embulk 0.8.29

```
embulk selfupdate
2017-08-10 12:54:10.851 +0900: Embulk v0.8.18
Checking the latest version...
Found new version 0.8.29.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.29.jar ...
Embulk::VERSION in (J)Ruby is deprecated. Use org.embulk.EmbulkVersion::VERSION instead. If this message is from a plugin, please tell this to the author of the plugin!
Updated to 0.8.29.
```

```
embulk preview config.yml.liquid
2017-08-10 12:54:37.897 +0900: Embulk v0.8.29
org.embulk.config.ConfigException: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:75)
	at org.embulk.config.DataSourceImpl.loadConfig(DataSourceImpl.java:220)
	at org.embulk.exec.PreviewExecutor.doPreview(PreviewExecutor.java:104)
	at org.embulk.exec.PreviewExecutor.access$000(PreviewExecutor.java:31)
	at org.embulk.exec.PreviewExecutor$1.run(PreviewExecutor.java:83)
	at org.embulk.exec.PreviewExecutor$1.run(PreviewExecutor.java:79)
	at org.embulk.spi.Exec.doWith(Exec.java:25)
	at org.embulk.exec.PreviewExecutor.preview(PreviewExecutor.java:79)
	at org.embulk.EmbulkEmbed.preview(EmbulkEmbed.java:169)
	at org.embulk.EmbulkRunner.previewInternal(EmbulkRunner.java:242)
	at org.embulk.EmbulkRunner.preview(EmbulkRunner.java:118)
	at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:469)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:100)
	at org.embulk.cli.EmbulkBundle.checkBundleWithEmbulkVersion(EmbulkBundle.java:42)
	at org.embulk.cli.EmbulkBundle.checkBundle(EmbulkBundle.java:15)
	at org.embulk.cli.Main.main(Main.java:26)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.TaskSerDe$TaskDeserializer.deserialize(TaskSerDe.java:181)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:3708)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2005)
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:72)
	... 15 more

Error: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
```

### configuration

config.yml.liquid

```
{% include 'commons/test' %}
out: {type: stdout}
```

commons/_test.yml.liquid

```yaml
#
in:
  type: file
  path_prefix: /private/tmp/hoge/csv/sample_
  decoders:
  - {type: gzip}
  parser:
    charset: UTF-8
    newline: LF
    type: csv
    delimiter: ','
    quote: '"'
    escape: '"'
    null_string: 'NULL'
    trim_if_not_quoted: false
    skip_header_lines: 1
    allow_extra_columns: false
    allow_optional_columns: false
    columns:
    - {name: id, type: long}
    - {name: account, type: long}
    - {name: time, type: timestamp, format: '%Y-%m-%d %H:%M:%S'}
    - {name: purchase, type: timestamp, format: '%Y%m%d'}
    - {name: comment, type: string}
```

I tested embulk 0.8.19 ~ 0.8.29.
This issue occurred on Embulk 0.8.28.

Test script.

```
% for i in `seq 19 29` ; do echo "******** VERSION 0.8.$i ********" ;  embulk selfupdate 0.8.$i ; embulk run config.yml.liquid ; done
```

```
% for i in `seq 19 29` ; do echo "******** VERSION 0.8.$i ********" ;  embulk selfupdate 0.8.$i ; embulk run config.yml.liquid ; done
******** VERSION 0.8.19 ********
2017-08-10 13:24:56.153 +0900: Embulk v0.8.28
Checking version 0.8.19...
Found version 0.8.19.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.19.jar ...
Updated to 0.8.19.
2017-08-10 13:25:28.402 +0900: Embulk v0.8.19
2017-08-10 13:25:30.727 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:25:30.731 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:25:30.736 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:25:30.801 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:25:30.812 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:25:30.949 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:25:30.955 +0900 [INFO] (main): Committed.
2017-08-10 13:25:30.955 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.20 ********
2017-08-10 13:25:33.066 +0900: Embulk v0.8.19
Checking version 0.8.20...
Found version 0.8.20.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.20.jar ...
Updated to 0.8.20.
2017-08-10 13:26:04.317 +0900: Embulk v0.8.20
2017-08-10 13:26:06.676 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:26:06.679 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:26:06.682 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:26:06.764 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:26:06.775 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:26:06.902 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:26:06.911 +0900 [INFO] (main): Committed.
2017-08-10 13:26:06.912 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.21 ********
2017-08-10 13:26:08.905 +0900: Embulk v0.8.20
Checking version 0.8.21...
Found version 0.8.21.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.21.jar ...
Updated to 0.8.21.
2017-08-10 13:26:43.096 +0900: Embulk v0.8.21
2017-08-10 13:26:45.335 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:26:45.338 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:26:45.341 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:26:45.411 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:26:45.419 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:26:45.544 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:26:45.552 +0900 [INFO] (main): Committed.
2017-08-10 13:26:45.555 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.22 ********
2017-08-10 13:26:47.550 +0900: Embulk v0.8.21
Checking version 0.8.22...
Found version 0.8.22.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.22.jar ...
Updated to 0.8.22.
2017-08-10 13:27:19.444 +0900: Embulk v0.8.22
2017-08-10 13:27:21.602 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:27:21.604 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:27:21.607 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:27:21.666 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:27:21.676 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:27:21.812 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:27:21.817 +0900 [INFO] (main): Committed.
2017-08-10 13:27:21.817 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.23 ********
2017-08-10 13:27:23.826 +0900: Embulk v0.8.22
Checking version 0.8.23...
Found version 0.8.23.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.23.jar ...
Updated to 0.8.23.
2017-08-10 13:27:51.411 +0900: Embulk v0.8.23
2017-08-10 13:27:53.429 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:27:53.432 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:27:53.435 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:27:53.479 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:27:53.486 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:27:53.589 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:27:53.593 +0900 [INFO] (main): Committed.
2017-08-10 13:27:53.594 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.24 ********
2017-08-10 13:27:55.583 +0900: Embulk v0.8.23
Checking version 0.8.24...
Found version 0.8.24.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.24.jar ...
Updated to 0.8.24.
2017-08-10 13:28:30.000 +0900: Embulk v0.8.24
2017-08-10 13:28:32.146 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:28:32.150 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:28:32.154 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:28:32.208 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:28:32.217 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:28:32.344 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:28:32.354 +0900 [INFO] (main): Committed.
2017-08-10 13:28:32.354 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.25 ********
2017-08-10 13:28:37.848 +0900: Embulk v0.8.24
Checking version 0.8.25...
Found version 0.8.25.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.25.jar ...
Updated to 0.8.25.
2017-08-10 13:29:21.162 +0900: Embulk v0.8.25
2017-08-10 13:29:23.577 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:29:23.580 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:29:23.582 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:29:23.615 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:29:23.621 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:29:23.734 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:29:23.739 +0900 [INFO] (main): Committed.
2017-08-10 13:29:23.740 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.26 ********
2017-08-10 13:29:28.643 +0900: Embulk v0.8.25
Checking version 0.8.26...
Found version 0.8.26.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.26.jar ...
Updated to 0.8.26.
2017-08-10 13:30:02.937 +0900: Embulk v0.8.26
2017-08-10 13:30:05.082 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:30:05.085 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:30:05.088 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:30:05.125 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:30:05.134 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:30:05.235 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:30:05.239 +0900 [INFO] (main): Committed.
2017-08-10 13:30:05.246 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.27 ********
2017-08-10 13:30:10.222 +0900: Embulk v0.8.26
Checking version 0.8.27...
Found version 0.8.27.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.27.jar ...
Updated to 0.8.27.
2017-08-10 13:30:37.070 +0900: Embulk v0.8.27
2017-08-10 13:30:39.474 +0900 [INFO] (0001:transaction): Listing local files at directory '/private/tmp/hoge/csv' filtering filename by prefix 'sample_'
2017-08-10 13:30:39.477 +0900 [INFO] (0001:transaction): "follow_symlinks" is set false. Note that symbolic links to directories are skipped.
2017-08-10 13:30:39.481 +0900 [INFO] (0001:transaction): Loading files [/private/tmp/hoge/csv/sample_01.csv.gz]
2017-08-10 13:30:39.526 +0900 [INFO] (0001:transaction): Using local thread executor with max_threads=8 / output tasks 4 = input tasks 1 * 4
2017-08-10 13:30:39.536 +0900 [INFO] (0001:transaction): {done:  0 / 1, running: 0}
1,32864,2015-01-27 19:23:49,20150127,embulk
2,14824,2015-01-27 19:01:23,20150127,embulk jruby
3,27559,2015-01-28 02:20:02,20150128,Embulk "csv" parser plugin
4,11270,2015-01-29 11:54:36,20150129,
2017-08-10 13:30:39.613 +0900 [INFO] (0001:transaction): {done:  1 / 1, running: 0}
2017-08-10 13:30:39.618 +0900 [INFO] (main): Committed.
2017-08-10 13:30:39.618 +0900 [INFO] (main): Next config diff: {"in":{"last_path":"/private/tmp/hoge/csv/sample_01.csv.gz"},"out":{}}
******** VERSION 0.8.28 ********
2017-08-10 13:30:44.610 +0900: Embulk v0.8.27
Checking version 0.8.28...
Found version 0.8.28.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.28.jar ...
Updated to 0.8.28.
2017-08-10 13:31:14.584 +0900: Embulk v0.8.28
org.embulk.config.ConfigException: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:75)
	at org.embulk.config.DataSourceImpl.loadConfig(DataSourceImpl.java:220)
	at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:536)
	at org.embulk.exec.BulkLoader.access$000(BulkLoader.java:35)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:391)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:387)
	at org.embulk.spi.Exec.doWith(Exec.java:25)
	at org.embulk.exec.BulkLoader.run(BulkLoader.java:387)
	at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:180)
	at org.embulk.EmbulkRunner.runInternal(EmbulkRunner.java:333)
	at org.embulk.EmbulkRunner.run(EmbulkRunner.java:175)
	at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:474)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:98)
	at org.embulk.cli.EmbulkBundle.checkBundleWithEmbulkVersion(EmbulkBundle.java:42)
	at org.embulk.cli.EmbulkBundle.checkBundle(EmbulkBundle.java:15)
	at org.embulk.cli.Main.main(Main.java:26)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.TaskSerDe$TaskDeserializer.deserialize(TaskSerDe.java:181)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:3708)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2005)
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:72)
	... 15 more

Error: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
******** VERSION 0.8.29 ********
2017-08-10 13:31:18.541 +0900: Embulk v0.8.28
Checking version 0.8.29...
Found version 0.8.29.
Downloading https://dl.bintray.com/embulk/maven/embulk-0.8.29.jar ...
Updated to 0.8.29.
2017-08-10 13:31:50.773 +0900: Embulk v0.8.29
org.embulk.config.ConfigException: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:75)
	at org.embulk.config.DataSourceImpl.loadConfig(DataSourceImpl.java:220)
	at org.embulk.exec.BulkLoader.doRun(BulkLoader.java:536)
	at org.embulk.exec.BulkLoader.access$000(BulkLoader.java:35)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:391)
	at org.embulk.exec.BulkLoader$1.run(BulkLoader.java:387)
	at org.embulk.spi.Exec.doWith(Exec.java:25)
	at org.embulk.exec.BulkLoader.run(BulkLoader.java:387)
	at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:180)
	at org.embulk.EmbulkRunner.runInternal(EmbulkRunner.java:330)
	at org.embulk.EmbulkRunner.run(EmbulkRunner.java:173)
	at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:472)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:100)
	at org.embulk.cli.EmbulkBundle.checkBundleWithEmbulkVersion(EmbulkBundle.java:42)
	at org.embulk.cli.EmbulkBundle.checkBundle(EmbulkBundle.java:15)
	at org.embulk.cli.Main.main(Main.java:26)
Caused by: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
	at org.embulk.config.TaskSerDe$TaskDeserializer.deserialize(TaskSerDe.java:181)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:3708)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2005)
	at org.embulk.config.ModelManager.readObjectWithConfigSerDe(ModelManager.java:72)
	... 15 more

Error: com.fasterxml.jackson.databind.JsonMappingException: Field 'in' is required but not set
 at [Source: N/A; line: -1, column: -1]
```

--- a/embulk-core/src/main/java/org/embulk/EmbulkRunner.java
+++ b/embulk-core/src/main/java/org/embulk/EmbulkRunner.java
@@ -417,7 +417,9 @@ public class EmbulkRunner
             return this.embed.newConfigLoader().fromYamlString(
                 runLiquid(new String(Files.readAllBytes(configFilePath), StandardCharsets.UTF_8),
                           templateParams,
-                          templateIncludePath));
+                          (templateIncludePath == null
+                           ? configFilePath.toAbsolutePath().getParent().toString()
+                           : templateIncludePath)));
         }
         else if (EXT_YAML.matcher(configFilePath.toString()).matches()) {
             return this.embed.newConfigLoader().fromYamlString(

